src/pages/steps/modals/PictureUploadModal.tsx

```diff
@@ -191,9 +213,29 @@ const PictureUploadModal = ({
   const image = watchedFile?.[0];
 
   useEffect(() => {
-    reset(imageToEdit ?? {});
+    const resetData = imageToEdit ?? {
+      description: '',
+      copyrightHolder: '',
+    };
+    reset(resetData);
   }, [imageToEdit, reset, visible]);
 
+  useEffect(() => {
+    setValue('file', draggedImageFile);
+  }, [draggedImageFile, setValue]);
+
+  const handleInternalOnDrop = (e: DragEvent<HTMLElement>) => {
+    e.nativeEvent.preventDefault();
+
+    if (!e) return;
```

Comment:
Shouldn't the early return be on the first line?
```suggestion
    if (!e) return;
 
    e.nativeEvent.preventDefault();
```